big_tests/dynamic_domains.spec

```diff
@@ -9,7 +9,7 @@
 %% http://www.erlang.org/doc/apps/common_test/run_test_chapter.html#test_specifications
 {include, "tests"}.
 
-{suites, "tests", domain_isolation_SUITE}.
+{suites, "tests", acc_e2e_SUITE}.
```

Comment:
why do we remove domain_isolation_SUITE?

Reply:
This suite was passing because `filter_local_packet` was executed per domain. I could update the test suite to use host types correctly, but the MUC test would fail. Can I do it in another PR?